utils/calculators.ts

```diff
@@ -165,3 +167,12 @@ export function DevMiningCalculator({
     },
   };
 }
+
+export function calculateFairValue(
+  fundingRate: BigNumber,
+  marketPrice: string
+) {
+  // convert market price to wei from a numerical string in eth
+  const marketPriceWei = parseEther(parseFloat(marketPrice).toFixed(18));
```

Comment:
Is it possible for `marketPrice` or `marketPriceWei.mul(fundingRate)` to not be in 18 precision?

Reply:
currently the marketprice is pulled from unis subgraph and it looks like this is always going to be in decimal form. it may be the case in the future though this assumption breaks. I will add a comment for it